common.gypi

```diff
@@ -10,7 +10,9 @@
     'component%': 'static_library',   # NB. these names match with what V8 expects
     'msvs_multi_core_compile': '0',   # we do enable multicore compiles, but not using the V8 way
     'python%': 'python',
-
+    'gas_version%': 0,
+    'llvm_version%': 0,
```

Comment:
What is this?

Reply:
It's a variable set in the [configure script](https://github.com/nodejs/node/blob/master/configure#L669) when clang is detected.

Reply:
I'm confused. The configure script wasn't modified in this PR. How come these two lines weren't necessary before?

Reply:
In order to check `llvm_version` version (without first doing "feature detection"), we need there vars with default value (90f055fdaf512644a6503ae940becd6ce1a9c2ae).

Reply:
P.S. It not necessary in the current changeset, so I could move this to a new PR